### DISCUSSION TITLE
Ignoring previously reported comments?

### DISCUSSION THREAD

**Discussion by thibaultmol** [ThioJoe/YT-Spammer-Purge#599] [General]
Am I missing something, but it seems like if you run the same scan again, it just highlights the comments again you previously reported. Is it not possible to just only show 'new' spam comments?

**Comment by Firecul** [comment]
That would be good to have a feature to save which logs have been reported so we don't do them again,especially if we are saving logs anyway.

**Comment by ThioJoe** [comment]
This could be a thing, but i'd have to see how it affects speed. You'd quickly get to the point of checking thousands of comment IDs each against a list of tens of thousands of comment IDs.

Although I guess it would be feasible to group the comment IDs by video ID first
